lib/rules/web/admin_console/4.6/overview.xyaml

```diff
@@ -216,4 +226,65 @@ group_by_label:
     dropdown_menu_item: <label>
   action: click_groupby_dropdown_button
   action: click_dropdown_menu_item
-
+click_workloads_tab:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: click
+send_key_to_workloads:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: send_keys <key>
+check_workload_selected:
+  elements:
+  - selector:
+      xpath: //li[contains(@class,'pf-c-data-list__item') and contains(@class,'pf-m-selected')]
+click_workload_details:
+  element:
+    selector:
+      xpath: //button[contains(text(),'Details')]
+    op: click
+check_workload_name:
+  elements:
+  - selector:
+      xpath: //div[@class='resource-overview__summary']//button[text()='Name']
```

Comment:
I tried on workloads page with //div[@class='resource-overview__summary']//button[text()='Name'], but could not find any element, is it suitable for 4.6?

Reply:
@yanpzhan Yes this is present in the side bar details overview pane that opens up when you select a workload. They converted it to a button from dt  back in 4.4